crates/mp4/src/lib.rs

```diff
@@ -1,3 +1,5 @@
+#![deny(unsafe_code)]
```

Comment:
I know a future pr will address mp4 so perhaps let's remove this for now and address it then.